pkgs/games/badlion-client/default.nix

```diff
@@ -2,11 +2,11 @@
 
 let
   pname = "badlion-client";
-  version = "3.15.0";
+  version = "4.0.0";
 
   src = fetchurl {
     url = "https://client-updates-cdn77.badlion.net/BadlionClient";
-    hash = "sha256-HqMgY9+Xnp4uSTWr//REZGv3p7ivwLX97vxGD5wqu9E=";
+    hash = "sha256-qI/5eNgaDgyLC9NOrHaDkITZsYsBLTOyqzGwobUDKh8=";
```

Comment:
the hash changed again, 
is there a stable URL that we can use ?

Reply:
I was trying to find it but now unfortunately it not, You know a way to fix this issue?

Reply:
Can we contact upstream?

Reply:
I guess we can but i think it not gonna help because launcher is design to automatic update